products/secretmanager/api.yaml

```
@@ -0,0 +1,163 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Secretmanager
```

```suggestion
name: SecretManager
```

I think `google_secret_manager_` is more in line with the naming convention we tend to use on new products. Is there something I'm missing that would make us prefer the `secretmanager` slug?

---

Thank you!  I had that originally then changed my mind after looking at a different service.  I'll change it back, as I like this better too.  Thanks!